go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
+	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
```

Comment:
IIRC, there's a medium CVE attached to x/net and we should double-check whether we're introducing an affected version.

Reply:
Is [this](https://www.cvedetails.com/cve/CVE-2021-33194/) the CVE you heard about?

Reply:
Yeah, that's the one I was thinking about - it looks like we're in the clear.

Reply:
Seems fine:
```
golang.org/x/net before v0.0.0-20210520170846-37e1c6afe023 allows attackers to cause a denial of service (infinite loop) via crafted ParseFragment input.
```